packages/mui-base/src/Slider/Slider.test.tsx

```diff
@@ -447,4 +447,144 @@ describe('<Slider />', () => {
       expect(secondThumb.getAttribute('data-active')).to.equal('false');
     });
   });
+
+  it('should support Shift + Left Arrow / Right Arrow keys', () => {
```

Comment:
Could you please also test up and down arrow functionality?

Reply:
Yeah, I will add test, I haven't realised we don't have one 👍